robottelo/helpers.py

```diff
@@ -280,6 +281,20 @@ def add_remote_execution_ssh_key(hostname, key_path=None, **kwargs):
     ssh.add_authorized_key(server_key, hostname=hostname, **kwargs)
 
 
+def add_remote_execution_capsule_to_subnet(hostname, capsule_id=1):
```

Comment:
I like helpers because you read them once and can continue reading other testing with the high level understand if it. Besides I agree on better naming it. I added a discussion about it on Friday mtg.